Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx

```diff
@@ -115,7 +121,7 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
   mOutputDecoderErrors.clear();
 
   if (isLostTimeframe(ctx)) {
-    sendData(ctx, mOutputCells, mOutputTriggerRecords, mOutputDecoderErrors);
+    sendData(ctx, mOutputCells, mOutputTriggerRecords, mOutputDecoderErrors, 0xdeadbeaf);
```

Comment:
Why do you need to send with 0xdeadbeaf subspec? Since the cell consumers (except the new event-builder) expect subspec 0, with this you will be obliged to always run the event-builder, even if the full processing is done on the EPN and event building is not needed.